lib/xcpretty/parser.rb

```diff
@@ -77,6 +77,10 @@ module Matchers
     # $2 file_name (e.g. Main.storyboard)
     COMPILE_STORYBOARD_MATCHER = /^CompileStoryboard\s(.*\/([^\/].*\.storyboard))/
 
+    # @regex Captured groups
+    # $1 file_path (e.g. Assets.xcassets)
+    COMPILE_ASSET_CATALOG_MATCHER = /^CompileAssetCatalog\s.*\s((?:\\.|[^ ])+\/(?:\\.|[^ ])+\.xcassets)/
```

Comment:
Right. Can you explain why `\\` is needed? It seems to me that `.` covers that up.

Reply:
This is because I want the group to end at the first space that isn't part of the path ('\\.' will match only spaces in the path, `.` might match the space after the path).